GitCommands/GitCommands.csproj

```diff
@@ -239,6 +241,9 @@
       <Install>true</Install>
     </BootstrapperPackage>
   </ItemGroup>
-  <Import Project="..\FxCop.targets" />
```

Comment:
All project imports of `FxCop.targets` were removed. The `<CodeAnalysisRuleSet>` above should replace these. And without removing this line, `GitExtensions.ruleset` was ignored (meaning projects referencing `FxCop.targets` wouldn't build).